nservicebus/messaging/third-party-integration.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Third Party Integration
+summary: Lists the necessary headers when receiving messages from non-NSB endpoints for better monitoring.
+tags:
+- Monitoring
+- Third Party Integration
+- ServiceInsight
+- ServicePulse
+related:
+- nservicebus/messaging/message-headers
+---
+
+# How to integrate external systems with NServiceBus?
+
+When an NServiceBus endpoint receives messages from external systems such as BizTalk, TIBCO, etc. directly, the message itself might not contain regular all information that NServiceBus provides. That information is necessary for proper serialization of the message, as well as better monitoring and troubleshooting experience.
+
+To get the best use of your tools ensure that the following information is included in the message sent by the third party endpoints. 
+
+### Required information
```

Comment:
this makes no sense with the prevous statement.

>To get the best use of your tools ensure

implies it is optional but this title is "Required"